src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -1210,30 +1211,28 @@ int fmi3_xml_get_has_start(fmi3_xml_parser_context_t *context, fmi3_xml_variable
     return hasStart;
 }
 
-static void fmi3_log_error_if_start_required(
-    fmi3_xml_parser_context_t *context,
-    fmi3_xml_variable_t *variable)
-{
+static void fmi3_log_error_if_start_required(fmi3_xml_parser_context_t* context, fmi3_xml_variable_t* variable) {
+    // causality checks
     if (variable->causality == fmi3_causality_enu_input) {
-        jm_log_error(context->callbacks,
```

Comment:
jm_log_error didn't really do the formatting correctly.

Tests for these errors are missing, but put on the to-do list.

Reply:
I didn't think about it too much and mostly took the format that was already there, but I agree with your formatting suggestion; changed it for all these cases in that function.